src/main/scala/com/itechart/ny_accidents/DatasetSplitApplication.scala

```diff
@@ -0,0 +1,18 @@
+package com.itechart.ny_accidents
+
+import com.google.inject.Guice
+import com.itechart.ny_accidents.constants.Configuration
+import com.itechart.ny_accidents.parse.AccidentsParser
+import com.itechart.ny_accidents.utils.FileWriterUtils
+
+object DatasetSplitApplication extends App {
+  val injector = Guice.createInjector(new GuiceModule)
+  val accidentsParser = injector.getInstance(classOf[AccidentsParser])
+
+  val (schema, datasets) = accidentsParser.splitDatasetByYear(Configuration.PATH_TO_DATASET_TO_SPLIT)
```

Comment:
Good place for pattern matching

Reply:
In which way should i use pattern matching here? I have to pass all of this values in the same time

Reply:
Something like that
```
accidentsParser.splitDatasetByYear(PATH_TO_DATASET_TO_SPLIT) match { case (schema, datasets) =>
      datasets.foreach { case (key, value) =>
        FileWriterUtils.createNewCsv(
          s"${Configuration.SPLIT_DATASET_BASE_FOLDER}/${Configuration.CSV_BASE_NAME}_$key.csv",
          value,
          schema)
      }
  }
```

Reply:
Also I think you can statically import constants to make code less verbose, but it's up to you

Reply:
Yes, thank you. Rewrited.